lib/transports/idbTransport.js

```diff
@@ -15,15 +15,18 @@
 // DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-const idbCall = (config, xmlInput, cb) => {
-  const {
-    dbconn, dbstmt, IN, CLOB, CHAR, SQL_ATTR_DBC_SYS_NAMING, SQL_FALSE,
-    // idb-connector is an optional dependency, since users may not use this transport
-    // thus we can't globally require it
-    // eslint-disable-next-line max-len
-    // eslint-disable-next-line global-require, import/no-extraneous-dependencies, import/no-unresolved
-  } = require('idb-connector');
+let idb = null;
 
+try {
+  // eslint-disable-next-line global-require
+  idb = require('idb-connector');
+} catch (e) {
+  if (e.code !== 'MODULE_NOT_FOUND') {
+    throw e;
+  }
+}
+
+const idbCall = (config, xmlInput, cb) => {
```

Comment:
This transport uses an arrow function instead of the `function` keyword. Any reason for the inconsistency?

Reply:
I think that this was a hold over from [istoredp.js](https://github.com/IBM/nodejs-itoolkit/commit/47b2a874d0ca3b787039d15e995aa674bcdaa88c#diff-e3a3f52c6d9adb9323a6bbad2b1c829d243d2d02f08b5f1945ef1855716df897R19). We can make this a standard function.

Reply:
looks like the [httpTransport](https://github.com/IBM/nodejs-itoolkit/blob/master/lib/transports/httpTransport.js#L20) also uses arrow function. We can make this function as well for consistency across the transports.